src/utils/markdown.rs

```diff
@@ -47,39 +53,51 @@ pub fn underline(s: &str) -> String {
 ///
 /// Passed string will not be automatically escaped because it can contain
 /// nested markup.
+#[must_use = "This function returns a new string, rather than mutating the argument, so calling it \
+              without using it's output does nothing useful"]
```

Comment:
"it's" is an abbreviation for "it is". In this context, you want to use "its".

Reply:
fixed